### PR TITLE
import/export gpx/kml

### PR DESCRIPTION
**Pull request by rmichaelis**
Could you please review.

I've added the missing echo service (exportgpxkml).
This service will be removed when c2cgeoportal will be updated.

### CODE REVIEW

**Comment by rmichaelis** [comment]
fixes #791

**Comment by rmichaelis** [comment]
Could you please review and test here : http://devv3.geoportail.lu/renaud

File: geoportailv3/static/js/mymaps/mymaps.html [hunk 1]
```diff
@@ -36,6 +36,24 @@
     <span class="map-owner" ng-show="ctrl.getMapOwner()">
       {{'Carte créée par'|translate}} {{ctrl.getMapOwner()}}
     </span>
+    <div ng-show="ctrl.showKml" class="alert alert-info">
+      <div><strong translate>Importer un kml</strong>
+      <button type="button" class="close" aria-label="Close" ng-click="ctrl.closeKmlPanel();"><span aria-hidden="true">&times;</span></button>
+      </div>
+      <span translate>
+        Veuillez sélectionner le fichier KML que vous désirer importer.
```
Comment (rmichaelis) [inline geoportailv3/static/js/mymaps/mymaps.html]: **désirez**

**Comment by jaykayone** [comment]
import de gpx et kml génériques est OK

**Comment by rmichaelis** [comment]
@jaykayone  @petzlux Could you please check if import is working fine on Mac ?

**Comment by geoportallux** [comment]
?i did that yesterday

________________________________

**Comment by petzlux** [comment]
will check as well soonish

**Comment by rmichaelis** [comment]
I just would like to know if you get the file selector when you click on the import menu item.

**Comment by geoportallux** [comment]
?works

________________________________

**Comment by geoportallux** [comment]
but there is a bug, when I create a new map, the bglayer goes to blank

________________________________

**Comment by petzlux** [comment]
the file selector appears correctly in my Mac. it works also to import a kml from google earth!

**Pull request merged**